backend/src/main/java/com/github/gadenko/gamereviewsite/backend/service/GameReviewService.java

```diff
@@ -50,4 +50,7 @@ public void deleteGameReview(String id) {
         gameReviewRepo.deleteById(id);
     }
 
+    public GameReview updateGameReviewById(GameReview updatedGameReview) {
+        return gameReviewRepo.save(updatedGameReview);
+    }
```

Comment:
Theoretisch könntest du hier auch noch mal wie in der add...-Methode überprüfen, was reinkommt, oder?